pkg/globalmanager/incrementallearningjob.go

```diff
@@ -365,6 +389,15 @@ func (jc *IncrementalJobController) updateIncrementalJobConditions(incrementaljo
 			} else {
 				// watch pod status, if pod running, set type running
 				newConditionType = sednav1.ILJobStageCondRunning
+				pod := jc.getSpecifiedPods(incrementaljob, string(jobStage))
+				ann := incrementaljob.Annotations
+				if ann == nil {
+					ann = make(map[string]string)
+				}
+				ann[string(jobStage)] = pod.Spec.NodeName
```

Comment:
please add a prefix something like `sedna.io` to the annotation key in case not conflict with user's keys.

Reply:
const AnnotationsKeyPrefix="sedna.io/"
ann[AnnotationsKeyPrefix + string(jobStage)] = pod.Spec.NodeName